### ISSUE TITLE
Incosistency between javadoc and code

### ISSUE THREAD

**Issue by GrafDiffusore** [resilience4j/resilience4j#1008]
Resilience4j version: 1.4.0

Java version: 8

In javadoc of `CircuitBreaker`:

```
 * The state of the CircuitBreaker changes from CLOSED to OPEN when the failure rate is above a
 * (configurable) threshold.
```

but from code (and tests I did) it seems that CLOSED->OPEN occurs if threshold is above *or equal*:
```
        if (failureRateInPercentage >= failureRateThreshold) {
            return Result.FAILURE_RATE_ABOVE_THRESHOLDS;
        }

```

Similar inconsistency seems to exist for transitions from HALF_OPEN state.

**Comment by RobWin** [comment]
Would you like to raise a PR?

**Comment by GrafDiffusore** [comment]
> Would you like to raise a PR?

Erm, I have no PR ready If it's still open next 2 weeks, I could help with that.

Plus, it is unclear -- which one should be fixed -- code or javadoc? :)

**Comment by RobWin** [comment]
Code is always correct :)

**Comment by dlsrb6342** [comment]
Fixed by #1011
